src/main/java/org/mskcc/smile/service/impl/ValidRequestCheckerImpl.java

```diff
@@ -63,6 +63,7 @@ public String getFilteredValidRequestJson(String requestJson) throws IOException
         // validate each sample json and add to validSampleList if it passes check
         Boolean isCmoRequest = isCmo(requestJsonMap);
         List<Object> updatedSampleList = new ArrayList<>();
+        List<Object> invalidRequestSamplesStatuses = new ArrayList<>();
         int validSampleCount = 0;
         for (Object sample: sampleList) {
             Map<String, Object> sampleMap = mapper.convertValue(sample, Map.class);
```

Comment:
Nit: maybe it's just my VS Code setup, but I'm seeing the warning `Type safety: The expression of type Map needs unchecked conversion to conform to Map<String,Object> Java(16777748)` on this line and a few other instances of `mapper.convertValue()` and `mapper.readValue()`. It seems like this error is [not concerning](https://stackoverflow.com/questions/53973126/type-safety-the-expression-of-type-map-needs-unchecked-conversion-to-conform), but in case you also see it and want to get rid of the warning messages, it looks like we can do so by specifying the Map type during conversion:
```
// before
Map<String, Object> sampleMap = mapper.convertValue(sample, Map.class);

// after
import com.fasterxml.jackson.core.type.TypeReference;
...
Map<String, Object> sampleMap = mapper.convertValue(sample, new TypeReference<Map<String, Object>>() {});
```